resources/views/site/digitalPublicationDetail.blade.php

```diff
@@ -45,17 +45,18 @@
             @component('components.molecules._m-title-bar', [
                 'variation' => 'm-title-bar--compact m-title-bar--light',
             ])
+                @slot('links', $topLevelArticle->children->count() > 0  && !$showAll ? ([['label' => 'Browse all '. $topLevelArticle->children->count(). ' '. $topLevelArticle->title, 'href' => route('collection.publications.digital-publications.showListing', ['id' => $item->id, 'slug' => $item->getSlug()]) . '#' . Str::kebab($topLevelArticle->title)]]) : '')
```

Comment:
This is a pretty long line, could it be moved into the presenter as a function?

Reply:
Good call.